_literate/EX-powergen/tutorial.jl

```diff
@@ -168,57 +160,35 @@ savefig(joinpath(@OUTPUT, "hist_wind.svg")) # hide
 # Finally, before fitting the estimator, we might want to gauge what to expect from them by looking at scatter plots.
 # Let's look at solar power first.
 
-fig = figure(figsize=(15, 15))
 
-subplot(221)
-scatter(data.Solar_gen, data.Radiation_dir)
-xlabel("Solar power (kW)", fontsize=14)
-ylabel("Solar radiation - directional", fontsize=14)
+p1 = scatter(data.Solar_gen, data.Radiation_dir, size=(150,150), legend=false, xlabel="Solar power (kW)", ylabel="Solar radiation - directional")
```

Comment:
let's put this over several lines
```suggestion
p1 = scatter(
    data.Solar_gen, 
    data.Radiation_dir, 
    size=(150,150), 
    legend=false, 
    xlabel="Solar power (kW)", 
    ylabel="Solar radiation - directional",
)
```

Ditto the other long lines below.

Reply:
Okay. One day they will make a VSCode extension for a Julia linter (I know there is a package for this as well).